include/crm/common/internal.h

```diff
@@ -236,6 +236,19 @@ pcmk__clear_flags_as(const char *function, int line, uint8_t log_level,
     return result;
 }
 
+/*!
+ * \internal
+ * \brief Get readable string for whether a flag is set
+ *
+ * \param[in] flag_group    Group of flags to check
+ * \param[in] flag          Flag to check whether set
+ *
+ * \return "true" if \p flag is set in \p flag_group, otherwise "false"
+ */
+#define pcmk__flag_str(flag_group, flag)    \
```

Comment:
I think a static inline function would work here. Ultimately we're calling `pcmk_all_flags_set()` (a function) with the same arguments. I think we both prefer inlines where possible, to avoid CPP quirks. Though this one should be fine.

Reply:
Good idea. I made it call `pcmk_all_flags_set()` directly to be clearer that giving multiple flags is possible.

Reply:
update doxygen when you do